package/MDAnalysis/transformations/fit.py

```diff
@@ -0,0 +1,206 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""\
+Fitting transformations --- :mod:`MDAnalysis.transformations.fit`
+=================================================================
+
+Translate and/or rotates the coordinates of a given trajectory to align
+a given AtomGroup to a reference structure.
+
+.. autofunction:: fit_translation
+
+.. autofunction:: fit_rot_trans
+
+"""
+from __future__ import absolute_import
+
+import numpy as np
+from functools import partial
+
+from ..analysis import align
+from ..lib.util import get_weights
+from ..lib.transformations import euler_from_matrix, euler_matrix
+
+
+def fit_translation(ag, reference, plane=None, weights=None):
+
+    """Translates a given AtomGroup so that its center of geometry/mass matches 
+    the respective center of the given reference. A plane can be given by the
+    user using the option `plane`, and will result in the removal of
+    the translation motions of the AtomGroup over that particular plane.
+    
+    Example
+    -------
+    Removing the translations of a given AtomGroup `ag` on the XY plane by fitting 
+    its center of mass to the center of mass of a reference `ref`:
+    
+    .. code-block:: python
+    
+        ag = u.select_atoms("protein")
+        ref = mda.Universe("reference.pdb")
+        transform = mda.transformations.fit_translation(ag, ref, plane="xy",
+                                                        center_of="mass")
```

Comment:
`center_of` is no longer a valid keyword argument to `fit_translation`, so the docstring needs updating